tools/internal_ci/linux/grpc_e2e_performance_v2.sh

```diff
@@ -77,6 +77,11 @@ buildConfigs() {
         -s big_query_table="${table}" -s timeout_seconds=900 \
         -s prebuilt_image_prefix="${PREBUILT_IMAGE_PREFIX}" \
         -s prebuilt_image_tag="${UNIQUE_IDENTIFIER}" \
+        -a buildNumber="${KOKORO_BUILD_NUMBER}" \
```

Comment:
I would prefix all these annotations with a single prefix, for instance "ci_", to indicate that they belong together logically.